pkg/aspect/printaction/printaction.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package printaction
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+
```

Comment:
nit: Blank line

Reply:
that one is intentional to separate our internal aspect.build imports and imports from 3rd party deps :)